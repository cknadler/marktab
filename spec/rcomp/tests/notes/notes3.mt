# note shorthand, single string
1:1 2 3 4
