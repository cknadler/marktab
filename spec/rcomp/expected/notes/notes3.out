e|-1-2-3-4-|
B|---------|
G|---------|
D|---------|
A|---------|
E|---------|

